pages/osx/base64.md

```diff
@@ -13,8 +13,8 @@
 
 - Encode from stdin:
 
-`echo -n {{plain_text}} | base64`
+`echo -n "{{plain_text}}" | base64`
 
 - Decode from stdin:
 
-`echo -n {{base64_text}} | base64 --decode`
+`echo -n "{{base64_text}}" | base64 --decode`
```

Comment:
I think a base64 text doesn't need `"`.

Reply:
Potentially this placeholder can contain text with spaces that's why I quoted it to ensure that no word splitting is performed.

Reply:
a base64 string doesn't have spaces.

Reply:
Thanks. It is my mistake. :)